File: [CODE]/rightoversJAVA/src/main/java/sep3/g3/rightoversjava/model/FoodPostReservationDto.java
package sep3.g3.rightoversjava.model;

public class FoodPostReservationDto {
    // TODO: Make join table when the user entity is made for the DB
    public int userId;
    public int foodPostId;

    public FoodPostReservationDto() {
    }

    public FoodPostReservationDto(int userId, int foodPostId) {
        this.userId = userId;
        this.foodPostId = foodPostId;
    }

    public int getUserId() {
        return userId;
    }

    public void setUserId(int userId) {
        this.userId = userId;
    }

    public int getFoodPostId() {
        return foodPostId;
    }

    public void setFoodPostId(int foodPostId) {
        this.foodPostId = foodPostId;
    }
}

File: [CODE]/rightoversBlazorNWEB/Domain/DTOs/FoodPostReservationDto.cs
namespace Domain.DTOs;

public class FoodPostReservationDto
{

    public FoodPostReservationDto(string username, int foodPostId)
    {
        Username = username;
        FoodPostId = foodPostId;
    }

    public string Username { get; set; }
    public int FoodPostId { get; set; }
}